incubator/cloudflare-datadog/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+description: A Helm chart for cloudflare-datadog service
```

Comment:
"Bridge between Cloudflare Zone Analytics API and Datadog Metrics." is a better description